pkg/questions/questions.go

```diff
@@ -137,6 +153,6 @@ func PromptOptional(text, def string) (string, error) {
 			answer = def
 		}
 
-		return answer, nil
+		return answer, nil //nolint:staticcheck
```

Comment:
This claims that is returning on the first loop iteration and I agree, but Im not sure I understand why this was done this way and not out of the loop. @mudler any ideas? Would it be safe to remove the loop here?

Reply:
it doesn't return always as if there is no selection it hits `continue`, so it would keep looping. On the other hand I agree, better have a break statement and returning at the end

Reply:
I think this was just copied from the `Prompt` function above which waits until it gets a proper response, but in this case, any response is valid so it should finish on the first try